tests/test_record.py

```diff
@@ -160,3 +164,18 @@ def test_path_extra(self):
             ''')
             spider.record(settings=dict(AUTOUNIT_EXTRA_PATH='abc'))
             spider.test()
+
+    def test_empty(self):
```

Comment:
So this isn't related to the issue fixed, but since `test_dotted_name` uses this test content, if autounit didn't handle this case it's possible `test_dotted_name` could fail even though the sanitization issue is fixed.  So I'm including this test as a "dependency" of the new test in a conceptual-tree sense.